packages/monosize/src/commands/compareReports.mts

```diff
@@ -25,7 +25,8 @@ async function compareReports(options: CompareReportsOptions) {
 
   const localReportStartTime = process.hrtime();
   const localReport = await collectLocalReport({
-    ...(options['report-files-glob'] && { reportFilesGlob: options['report-files-glob'] }),
```

Comment:
this short-circuiting with spread operation gave me hard time what is actually happening so I simplified it